testsuite/MDAnalysisTests/analysis/conftest.py

```diff
@@ -91,11 +93,33 @@ def client_RMSF(request):
     return request.param
 
 
+# MDAnalysis.analysis.dihedrals
+
+@pytest.fixture(scope='module', params=params_for_cls(Dihedral))
+def client_Dihedral(request):
+    return request.param
+
+
+@pytest.fixture(scope='module', params=params_for_cls(Ramachandran))
+def client_Ramachandran(request):
+    return request.param
+
+
+@pytest.fixture(scope='module', params=params_for_cls(Janin))
+def client_Janin(request):
+    return request.param
+
+
+# MDAnalysis.analysis.gnm
+  
 @pytest.fixture(scope='module', params=params_for_cls(GNMAnalysis))
 def client_GNMAnalysis(request):
     return request.param
 
-  
+
+# MDAnalysis.analysis.bat
+
 @pytest.fixture(scope='module', params=params_for_cls(BAT))
 def client_BAT(request):
-  return request.param
\ No newline at end of file
+  return request.param
+
```

Comment:
```suggestion
    return request.param
```

Reply:
Make darker happy.

[Line 124:3](https://github.com/MDAnalysis/mdanalysis/blob/7cdde28719ad26c2b9dbd7bfae5a2387934d6119/testsuite/MDAnalysisTests/analysis/conftest.py#L124): [E111](https://duckduckgo.com/?q=pep8%20E111) indentation is not a multiple of four
[Line 125:1](https://github.com/MDAnalysis/mdanalysis/blob/7cdde28719ad26c2b9dbd7bfae5a2387934d6119/testsuite/MDAnalysisTests/analysis/conftest.py#L125): [W391](https://duckduckgo.com/?q=pep8%20W391) blank line at end of file